.github/workflows/ci.yml

```
@@ -0,0 +1,19 @@
+name: CI
+on:
+  push:
+  pull_request:
+env:
+  CARGO_TERM_COLOR: always
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+    steps:
+    - run: rustc -vV
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --verbose
```

```suggestion
      run: cargo check --verbose
```
Do we even need to build at all? I'd suggest either changing `build` or adding separate `check` to make the CI feedback fast.

---

The artifacts built are used by `cargo test` below - oftentimes it's split like that to discern between failure to build and failure when testing and that's also in the default Rust actions template that I based on.

Do you want me to skip the `cargo build --verbose` step altogether?

---

No, I think what you're saying totally makes sense :)